test/clitest.hs

```diff
@@ -136,50 +136,6 @@ main = do
         let hexStr = Types.bsToHex c
         (_, stdout, _) <- readProcessWithExitCode "cabal" ["run", "exe:hevm", "--", "symbolic", "--solver", "empty", "--code", hexStr] ""
         stdout `shouldContain` "SMT solver says: Result unknown by SMT solver"
-
```

Comment:
Why are we removing this test?

Reply:
The inequality solves this immediately, so the UNSAT cache is never activated...

Reply:
It was a rather flaky test anyway. It depended on the ordering of things. Most of the time it worked. I'd rather remove anyway.